frontend/src/shared/helpers.tsx

```diff
@@ -730,6 +730,16 @@ export const CohortSampleDetailsColumns: ColDef[] = [
     headerTooltip: 'User who last updated the "Billed" status',
     headerComponentParams: createCustomHeader(lockIcon + toolTipIcon),
   },
+  {
+    field: "custodianInformation",
+    headerName: "Data Custodian",
+    editable: true,
+  },
+  {
+    field: "accessLevel",
+    headerName: "Access Level",
+    editable: true,
```

Comment:
In our current setup, editability is enabled by adding the editable field names to the `editableSampleFields` array (currently at [line 791](https://github.com/mskcc/smile-dashboard/pull/130/files#diff-db966dd79103f32fddb49dcefcfba78d4325506ce364cd0e24779a3521aa7173R791)) instead of enabling it at the individual column definition level like here. It appears these fields are not editable right now in dev, at least on my end.

For next step, we should be good to go after removing `editable: true` here and adding `custodianInformation` and `accessLevel` to the `editableSampleFields` array.

(For context, this configuration currently happens at [lines 510-515](https://github.com/mskcc/smile-dashboard/pull/130/files#diff-db966dd79103f32fddb49dcefcfba78d4325506ce364cd0e24779a3521aa7173R510-R515). We do this to [1] save ourselves from manually adding `editable: true/false` to every column definition and [2] make clearer at a quick glance which fields we allow users to edit.)